src/main/java/ca/corefacility/bioinformatics/irida/database/changesets/AutomatedAnalysisToTemplate.java

```diff
@@ -0,0 +1,185 @@
+			UUID sistrId = sistrWorkflow.getWorkflowIdentifier();
+
+			//get the default params
+			List<IridaWorkflowParameter> defaultSistrParams = sistrWorkflow.getWorkflowDescription()
+					.getParameters();
+
+			//insert the sistr entries without metadata
+			insertWorkflow(jdbcTemplate, "Automated SISTR Typing", false, sistrId, "p.sistr_typing_uploads='AUTO'",
+					defaultSistrParams);
+			//insert the sistr entries with metadata
+			insertWorkflow(jdbcTemplate, "Automated SISTR Typing", true, sistrId,
+					"p.sistr_typing_uploads='AUTO_METADATA'", defaultSistrParams);
+		} catch (IridaWorkflowNotFoundException e) {
+			logger.warn("SISTR workflow not found.  Automated SISTR will not be converted to analysis templates.");
+			//Note this will definitely happen in the galaxy CI tests as only SNVPhyl and a test workflow are configured.
+		}
+
+		return new SqlStatement[0];
+	}
+
+	private void insertWorkflow(JdbcTemplate jdbcTemplate, String name, boolean updateSamples, UUID workflowId,
+			String where, List<IridaWorkflowParameter> params) {
+		/*
+		 * we're borrowing the 'automated' flag here to mark entries we need to add params to later.  at this point
+		 * there'll be nothing with a '1' in the automated flag.  we'll clear it later.
+		 */
+
+		//first get the project ids that have an automated submission
+		String idSql = "SELECT p.id FROM project p WHERE " + where;
+		List<Long> projectIds = jdbcTemplate.queryForList(idSql, Long.class);
+
+		//build the params for the insert submission
+		List<Object[]> queryParams = projectIds.stream()
+				.map(p -> {
+					return new Object[] { name, updateSamples, workflowId.toString(), p };
+				})
+				.collect(Collectors.toList());
+
+		//then insert the submisisons for each project
+		String submissionInsert = "INSERT INTO analysis_submission (DTYPE, name, created_date, priority, update_samples, workflow_id, submitter, submitted_project_id, automated) VALUES ('AnalysisSubmissionTemplate', ?, now(), 'LOW', ?, ?, 1, ?, 1)";
+		int[] updates = jdbcTemplate.batchUpdate(submissionInsert, queryParams);
+
+		//check if we did any updates
+		int update = IntStream.of(updates)
+				.sum();
+
+		//if we added anything, add the params
+		if (update > 0) {
+
+			// Insert the default params for the analysis type
+			for (IridaWorkflowParameter p : params) {
+
+				//first get the analysis submission ids we're inserting for
+				String paramSelect = "SELECT a.id FROM analysis_submission a WHERE a.name=? AND a.automated=1";
+				List<Long> submissionIds = jdbcTemplate.queryForList(paramSelect, Long.class, name);
+
+				//build the argument list for the query
+				List<Object[]> submissionParamArgs = submissionIds.stream()
+						.map(i -> {
+							return new Object[] { i, p.getName(), p.getDefaultValue() };
+						})
+						.collect(Collectors.toList());
+
+				//then insert the params for each submission
+				String paramInsert = "INSERT INTO analysis_submission_parameters (id, name,value) VALUES (?, ?, ?)";
```

Comment:
Same comment here. There's no audit tables being created for `analysis_submission_parameters`.